Include/cpython/frameobject.h

```diff
@@ -27,3 +27,9 @@ PyAPI_FUNC(int) _PyFrame_IsEntryFrame(PyFrameObject *frame);
 
 PyAPI_FUNC(int) PyFrame_FastToLocalsWithError(PyFrameObject *f);
 PyAPI_FUNC(void) PyFrame_FastToLocals(PyFrameObject *);
```

Comment:
Add `Py_DEPRECATED(3.18)` to these two APIs and to `PyFrame_LocalsToFast()` above.

Reply:
Scratch this comment, since the SC asked that the deprecation timelines be left unspecified for now.